experimental/linting_tool/lib/pages/saved_lints_page.dart

```diff
@@ -45,7 +45,7 @@ class SavedLintsPage extends StatelessWidget {
               ),
               itemCount: profilesStore.savedProfiles.length,
               cacheExtent: 5,
-              itemBuilder: (context, index) {
+              itemBuilder: (itemBuilderContext, index) {
```

Comment:
I'm curious why the `context` rename here

Reply:
I'm passing the `context` to a `ScaffoldMessenger`(to show a SnackBar) later in the code. If it isn't renamed, this context gets passed which causes the following error:
```
Unhandled Exception: Looking up a deactivated widget's ancestor is unsafe.
At this point the state of the widget's element tree is no longer stable.
To safely refer to a widget's ancestor in its dispose() method, save a reference to the ancestor by calling dependOnInheritedWidgetOfExactType() in the widget's didChangeDependencies() method.
```

Reply:
I'll look more closely at this after it is landed.